superbench/config/amd_mi100_hpe.yaml

```diff
@@ -40,16 +40,14 @@ superbench:
     rccl-bw:
```

Comment:
Would you please revise the description?

Reply:
updated